server/router.js

```diff
@@ -196,21 +196,31 @@ router.get('/api/group/:groupId/articles', (req, res) => {
 /*
 Get articles posted in a group as pages. Request should include query:
   limit: number of items per page
-  page: number of page to be loaded
+  page: number of page to be loaded (starting at 0)
   sort: field to sort on, must be field on Article model
   sort_dir: direction to sort in, -1 for decreasing, 1 for increasing
 */
 router.get('/api/group/:groupId/articles/paginated', (req, res) => {
-  const conditions = { query: {}, pagination: {}, sort: {} };
-  conditions.query.group = req.params.groupId;
-  if (req.query.limit) {
-    conditions.pagination.limit = req.query.limit;
+  const conditions = { pagination: {}, sort: {} };
+
+  // defaults
+  let limit = Number.parseInt(req.query.limit, 10);
+  if (!limit || limit < 0) {
+    limit = 50;
   }
```

Comment:
so, I didn't add any defaults at first with the idea that you could at some point use this same endpoint to retrieve paginated and non-paginated results... but this is good for now because obviously thats not how we're currently using it.